client/src/main/java/tds/config/ClientSystemFlag.java

```diff
@@ -0,0 +1,115 @@
+package tds.config;
+
+import java.sql.Timestamp;
+
+/**
+ Represents a record in the {@code configs.client_systemflags} table.
+ */
+public class ClientSystemFlag {
+    private String auditObject = "";
+    private String clientName = "";
+    private Boolean isPracticeTest;
+    private Boolean isOn;
+    private String description;
+    private Timestamp dateChanged;
+    private Timestamp datePublished;
+
+    public ClientSystemFlag() { }
+
+    public ClientSystemFlag(String auditObject, String clientName) {
+        this.setAuditObject(auditObject);
+        this.setClientName(clientName);
+    }
+
+    public String getAuditObject() {
+        return auditObject;
+    }
+
+    /**
+     * Protect against a {@link NullPointerException} in the event {@code equals} is called.  Possible code smell; look
+     * into refactoring {@code equals} method..
+     */
+    public void setAuditObject(String auditObject) {
+        this.auditObject = auditObject == null
+                ? ""
+                : auditObject;
+    }
```

Comment:
Huh - I'm having trouble remembering why I did this.  According to the schema, `auditObject` and `clientName` can never be `NULL`.  They're both members of the primary key, so they'll never ever be `NULL`.  But I must've run into it at some point; maybe with junky test data.  I'll double-check my old unit tests; I think that's the only way this condition could occur...